plugins/woocommerce-blocks/tests/e2e/tests/product-collection/product-collection.block_theme.spec.ts

```diff
@@ -803,6 +803,7 @@ test.describe( 'Product Collection', () => {
 				canvas: 'edit',
 			} );
 
+			await editor.enterEditMode();
```

Comment:
If the the `visitSiteEditor` has the `canvas: 'edit'` param set, it will load the entity in edit mode, which might cause a conflict with `enterEditMode()`. I'd generally recommend using `enterEditMode()` for now, since it also waits until the editor is loaded (meaning that I would just remove the `canvas: 'edit'` part 😄).